src/main/resources/org/jenkinsci/plugins/workflow/steps/scm/GenericSCMStep/help.jelly

```diff
@@ -2,11 +2,13 @@
 <?jelly escape-by-default='true'?>
 <div xmlns:j="jelly:core" xmlns:st="jelly:stapler">
     <p>
-    This is a special step that allows to run checkouts using any configuration
+    This is a special step that will checkout using the configuration
     options offered by any Pipeline-compatible SCM plugin.
-    To use a concrete SCM implementations, just install the corresponding plugin 
-    and check if it is shown in the list below.
-    Then select the SCM to use from the dropdown list and configure it as needed.
+    Install the corresponding SCM plugin and select it from the list in the
+    <strong><a href="https://jenkins.io/doc/book/pipeline/getting-started/#snippet-generator">Pipeline Snippet Generator</a></strong>.
```

Comment:
Oh, I just remembered that we should probably create a redirect link to use here instead of using a direct link in case of future changes. Maybe `https://jenkins.io/redirect/pipeline-snippet-generator`?

Reply:
See jenkinsci/workflow-scm-step-plugin#42.